lib/Nugget.js

```diff
@@ -23,29 +25,29 @@ import ScatterGraph from './views/ScatterGraph';
 import TrendLine from './views/TrendLine';
 import AreaGraph from './views/AreaGraph';
 
```

Comment:
This gets built for consumption as an ES6 module, so this style of export is necessary to have `Nugget.Events` instead of `Nugget.default.Events`.